Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -86,6 +86,12 @@ void MenuBar::AddToolsMenu()
   QMenu* tools_menu = addMenu(tr("Tools"));
   m_wad_install_action = tools_menu->addAction(tr("Install WAD..."), this, SLOT(InstallWAD()));
 
+  // Label will be set by a NANDRefresh later
+  m_boot_sysmenu = tools_menu->addAction(QStringLiteral(""), [this] { emit BootWiiSystemMenu(); });
+  m_boot_sysmenu->setEnabled(false);
+
+  connect(&Settings::Instance(), &Settings::NANDRefresh, [this] { UpdateToolsMenu(false); });
```

Comment:
I get why this is in Settings, but I don't think this should be part of it... This just doesn't seem right. At the same time I don't know enough about Qt or how DolphinQt2 is organised to suggest something better, so if no one can figure out a better way (or if people think it's fine), then feel free to ignore this comment.